core/editor/src/frontend/frontend_message_handler.rs

```diff
@@ -11,6 +11,7 @@ pub enum FrontendMessage {
 	ExpandFolder { path: Vec<LayerId>, children: Vec<LayerPanelEntry> },
 	SetActiveTool { tool_name: String },
 	UpdateCanvas { document: String },
+	UpdateDocumentTransform { transform: String },
```

Comment:
What was the reasoning for  returning a string vs 4 floats? returning floats seems cleaner, but I might very well be wrong on that one.

Reply:
I second this. Let's keep frontend messages frontend-agnostic. This string seems very SVG specific.